app/screens/settings/notification_settings_mobile/notification_settings_mobile_base.js

```diff
@@ -52,8 +89,15 @@ export default class NotificationSettingsMobileBase extends PureComponent {
     saveUserNotifyProps = () => {
         const props = {...this.state};
 
+        Reflect.deleteProperty(props, 'defaultSound');
```

Comment:
Maybe it would be easier to just select the fields we want to save instead of deleting the ones we don't?